packages/fluentui/e2e/tests/popupDismissScroll.spec.ts

```diff
@@ -25,4 +25,32 @@ describe('Popup - dismiss on scroll container', () => {
     cy.get('body').trigger('wheel'); // page scroll
     cy.notExist(`#${selectors.dismissScrollPopup.contentId}`); // popup dismissed
   });
+
+  describe('nested popup on context', () => {
+    const openNestedPopups = () => {
+      cy.get(`#${selectors.nestedPopup.parentPopupTriggerId}`).rightclick(); // opens popup 1
+      cy.visible(`#${selectors.nestedPopup.childPopupTriggerId}`); // popup 1 visible
+
+      cy.get(`#${selectors.nestedPopup.childPopupTriggerId}`).click(); // opens popup 2
+      cy.visible(`#${selectors.nestedPopup.childPopupContentId}`); // nested popup 2 visible
+    };
+
+    it('dismiss on scroll document', () => {
+      openNestedPopups();
+
+      cy.get('body').trigger('wheel'); // page scroll
```

Comment:
just being paranoid, did you check that this test fails without your change ?

Reply:
Good reminder and yes I did :)